examples/ThemeMaker.py

```diff
@@ -0,0 +1,271 @@
+        'progressbar_border': [null, null],
+        'slider': [null, null],
+        'slider_progress': [null, null],
+        'slider_button': [null, null],
+        'slider_button_hover': [null, null],
+        'switch': [null, null],
+        'switch_progress': [null, null],
+        'switch_button': [null, null],
+        'switch_button_hover': [null, null],
+        'optionmenu_button': [null, null],
+        'optionmenu_button_hover': [null, null],
+        'combobox_border': [null, null],
+        'combobox_button_hover': [null, null],
+        'dropdown_color': [null, null],
+        'dropdown_text': [null, null],
+        'dropdown_hover': [null, null],
+        'scrollbar_button': [null, null],
+        'scrollbar_button_hover': [null, null]
+    },
+    "text": {
+    "macOS": {
+        "font": "SF Display",
+        "size": -13
+        },
+    "Windows": {
+        "font": "Roboto",
+        "size": -13
+    },
+    "Linux": {
+        "font": "Roboto",
+        "size": -13
+        }
+    },
+    "shape": {
+        "button_corner_radius": 8,
+        "button_border_width": 0,
+        "checkbox_corner_radius": 7,
+        "checkbox_border_width": 3,
+        "radiobutton_corner_radius": 1000,
+        "radiobutton_border_width_unchecked": 3,
+        "radiobutton_border_width_checked": 6,
+        "entry_border_width": 2,
+        "frame_corner_radius": 10,
+        "frame_border_width": 0,
+        "label_corner_radius": 0,
+        "progressbar_border_width": 0,
+        "progressbar_corner_radius": 1000,
+        "slider_border_width": 6,
+        "slider_corner_radius": 8,
+        "slider_button_length": 0,
+        "slider_button_corner_radius": 1000,
+        "switch_border_width": 3,
+        "switch_corner_radius": 1000,
+        "switch_button_corner_radius": 1000,
+        "switch_button_length": 0,
+        "scrollbar_corner_radius": 1000,
+        "scrollbar_border_spacing": 4
+    }
+}
+
+
+#list of widgets
+Widgets=['Window', 'Frame', 'Button', 'CheckBox', 'Entry', 'Label', 'Text', 'ProgressBar', 'Slider',
+         'Switch', 'Menu + Dropdown', 'Scrollbar']
+
+#List containing menu content
+list0=['window_bg_color']
+list1=['frame_border', 'frame_low', 'frame_high']
+list2=['button','button_hover','button_border']
+list3=['checkbox_border','checkmark']
+list4=['entry','entry_border','entry_placeholder_text']
+list5=['label']
+list6=['text','text_disabled','text_button_disabled']
+list7=['progressbar','progressbar_progress','progressbar_border']
+list8=['slider','slider_progress','slider_button','slider_button_hover']
+list9=['switch','switch_progress','switch_button','switch_button_hover']
+list10=['optionmenu_button','optionmenu_button_hover','combobox_border','combobox_button_hover',
+        'dropdown_color','dropdown_hover','dropdown_text']
+list11=['scrollbar_button','scrollbar_button_hover']
+
+#Function for changing the values of menu
+def changemenu():
+    if current=='Window':

```

Comment:
If you have a dictionary like described above, you don't need all the if else statements, just use `widget_attributes[current]` or something similar

Reply:
Done